Documentation/guides/messages/README.md

```diff
@@ -19,7 +19,9 @@
 
 + [XA0000](xa0000.md): Could not determine $(AndroidApiLevel) or $(TargetFrameworkVersion).
 + [XA0001](xa0001.md): Invalid or unsupported `$(TargetFrameworkVersion)` value.
-+ [XA0118](xa0002.md): Could not find mono.android.jar
++ [XA0002](xa0002.md): Could not find mono.android.jar
```

Comment:
not sure why we had the wrong code here... Also XA0002 does not appear to be used anywhere in the repo...